common/amazon_support.py

```diff
@@ -15,6 +15,8 @@
 class AmazonItemCondition(Enum):
     # See https://sellercentral.amazon.com/gp/help/external/200386310?language=en_US&ref=efph_200386310_cont_G1831
     New = 10
+    Nuevo = 10
+    Neuf = 10
```

Comment:
```suggestion
    Neuf = 10
    Neu = 10
    Nuovo = 10
```

This adds German and Italian, too.